src/Core/src/Platform/Android/EntryExtensions.cs

```diff
@@ -51,5 +51,13 @@ internal static void SetInputType(this EditText editText, IEntry entry)
 			if (entry.IsPassword && ((editText.InputType & InputTypes.ClassNumber) == InputTypes.ClassNumber))
 				editText.InputType |= InputTypes.NumberVariationPassword;
 		}
+
+		public static void UpdatePlaceholder(this EditText editText, IEntry entry)
+		{
+			if (editText.Hint == entry.Placeholder)
```

Comment:
Is this really worth the extra interop. If we are different, then we have to set. If we are the same, then it is equivalent to setting it.

Reply:
I see you added this in a special commit. What was the reason for this? What do you know that I don't? XD

Reply:
I seen this in EntryRenderer for XF 😀 and it made sense to me to not assign them if they already have same value.

Reply:
OK, let's leave it as the renderer and be safe.